Allow en_GB as a local for KR and TW

SnomeaD opened the pull request:
Since it's working with the API:
https://tw.api.battle.net/wow/realm/status?locale=en_GB&apikey=***
It could be useful to have the choice to retrieve, for example, the server name in English.

benweier commented:
Thanks @SnomeaD this is undocumented so I successfully tested the KR and TW endpoints with `en_US` as well. I'll be happy to merge this with both English options in the allowed locales list.

SnomeaD commented:
Hello @benweier , thanks for the suggestion, I've added the en_US locale (and squashed my commits). And thanks for your work on this library, this is awesome!

benweier commented:
Thanks @SnomeaD v1.6.1 will be published soon with this change
